manager/state/raft/testutils/testutils.go

```diff
@@ -295,13 +295,14 @@ func NewNode(t *testing.T, clockSource *fakeclock.FakeClock, tc *cautils.TestCA,
 
 	keyRotator := NewSimpleKeyRotator(raft.EncryptionKeys{CurrentDEK: []byte("current")})
 	newNodeOpts := raft.NodeOptions{
-		ID:             securityConfig.ClientTLSCreds.NodeID(),
-		Addr:           l.Addr().String(),
-		Config:         cfg,
-		StateDir:       stateDir,
-		ClockSource:    clockSource,
-		TLSCredentials: securityConfig.ClientTLSCreds,
-		KeyRotator:     keyRotator,
+		ID:               securityConfig.ClientTLSCreds.NodeID(),
+		Addr:             l.Addr().String(),
+		Config:           cfg,
+		StateDir:         stateDir,
+		ClockSource:      clockSource,
+		TLSCredentials:   securityConfig.ClientTLSCreds,
+		KeyRotator:       keyRotator,
+		DisableStackDump: true,
```

Comment:
I don't think this situation has come up before with the raft unit tests - there is just not much concurrent store usage involved. We'll still get the stack traces within integration tests, where I think an issue would be far more likely.

But yeah, I can make this specific to `TestRaftWedgedManager`.